File: code/01问好/Go/01.go
/*
   2017-04-26 by Jimmy Xiang
 */

package main
import "fmt"


func main() {

    var name string

    fmt.Printf("What is your name?")

    fmt.Scanf("%s", &name)

    fmt.Printf("Hello, %s, nice to meet you!\n", name)

}

File: code/01问好/C/01.c
/*
   2017-04-26 by Jimmy Xiang
 */

#include <stdio.h>

 int main()
 {

    char name[128]={0};

    printf("What is your name?");
    scanf("%s",name);   //stackoverflow
    printf("Hello, %s, nice to meet you!\n", name);

    return 0;
 }

